fauna/response.py

```diff
@@ -50,44 +53,12 @@ def __init__(
         self._status_code = status_code
         self._traceparent = headers.get("traceparent", "")
         self._headers = headers
-
-        if "data" not in response_json:
-            raise ProtocolError(
-                self._status_code,
-                "Unexpected response",
-                f"Key 'data' not found in response body: \n{response_json}",
-            )
+        self._stats: Mapping[Union[str, Stat], Any] = {}
```

Comment:
can we assume the stats are ints?

Reply:
each stat has an individual data type declared, so this felt safer to me.